### DISCUSSION TITLE
Make break-word in Cell Table.

### DISCUSSION THREAD

**Discussion by aswinabbas** [heroui-inc/heroui#1045] [Help]
I wanna make a cell table with break-word and I implemented css props in the Table.Cell component, but it didn't work.

`<Table.Cell css={{ worBreak: "break-word" }} >longlonglonglong</Table.Cell>`

any one has solved this problem?

**Comment by susyawan** [comment]
````
<Table css={{tableLayout: "fixed"}}>
...
<Table.Cell css={{verticalAlign: "top", whiteSpace: "break-spaces"}}>...</Table.Cell>
...
</Table>
````

maybe
